### PR TITLE
Slight expansion on Javascript Section about the DOM

### PR DESCRIPTION
**Pull request by Kelketek**
@jzoldak It was requested that I add a bit of text about event listeners, mostly to help jog the thought process of the reader to help them figure out what they may have done wrong. This PR adds a little blurb for that. It will be reviewed by one the other OpenCraft folks first, and then we'll ping you again.

I'm not sure if there's a special format for contributing to this repository as there is for edx-platform. Let me know if you need this description to conform to a specific format.

I've also added myself to the AUTHORS file in this branch, though I'm not sure if that's needed just for docs.

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @Kelketek! It looks like you're a member of a company that does contract work for edX. If you're doing this work as part of a paid contract with edX, you should talk to edX about who will review this pull request. If this work is not part of a paid contract with edX, then you should ensure that there is an OSPR issue to track this work in [JIRA](https://openedx.atlassian.net), so that we don't lose track of your pull request. 

To automatically create an OSPR issue for this pull request, just visit this link: http://openedx-webhooks.herokuapp.com/github/process_pr?repo=edx%2Fbok-choy&number=79

File: docs/guidelines.rst [hunk 1]
```diff
@@ -89,7 +89,7 @@ Each test can then access the section title and assert that it matches what the
 Never use ``time.sleep()``
 --------------------------
 
-Sometimes, tests fail because when they check the page too soon.  Often, tests must wait for JavaScript on the page to finish manipulating the DOM.  In these cases, it is tempting to insert an explicit wait using ``time.sleep()``.  For example:
+Sometimes, tests fail because when they check the page too soon.  Often, tests must wait for JavaScript on the page to finish manipulating the DOM, such as when elements or even attaching event listeners.  In these cases, it is tempting to insert an explicit wait using ``time.sleep()``.  For example:
```
Comment (bradenmacdonald) [inline docs/guidelines.rst]: I think you a word. ("such as when _____ elements or")

Comment (Kelketek) [inline docs/guidelines.rst]: You're right. I did a word. I'll fix that.

**Comment by Kelketek** [comment]
@jzoldak , should be ready for you, or whoever is delegated to look at docs for this project.

**Comment by benpatterson** [comment]
:+1: 
@clytwynec could you take a look?

**Comment by clytwynec** [comment]
The change looks good to me :+1:

**Comment by benpatterson** [comment]
Thanks @Kelketek !

**Pull request merged**